torch/csrc/lazy/core/lazy_graph_executor.cpp

```diff
@@ -866,40 +839,6 @@ LazyGraphExecutor::ComputationCache::TypePtr LazyGraphExecutor::
 typedef SSIZE_T ssize_t;
 #endif
 
-void LazyGraphExecutor::BuildInputOutputAliases(
```

Comment:
Is this not something that is needed in PyTorch/XLA? Why is it being removed? CC: @JackCaoG

Reply:
oh yea, I missed this. There has been some changes on our end of `BuildInputOutputAliases` since we now wrpas input parameter if it is too big. I think we can keep it for now but will need to circle back and update this.

Reply:
Hmm... Yea, I just remember TSBackend never uses this and the XLA one has diverged a lot already. @antoniojkim Does MLIR use this? If not, I don't know what to keep there? Upstream the latest XLA implementation?

Reply:
We're not currently using this in Torch-MLIR. We found that the functionalization pass broke this function's ability to properly map the inputs to the outputs. So, we are currently using a custom MLIR specific solution. If PyTorch/XLA is not using it, we can remove it.

Can you point me to the latest XLA implemention? Maybe its something we can use in Torch-MLIR as well if its upstreamed

Reply:
This is interesting, we need to make sure input_output_aliasing works, otherwise we will take a huge hit on memory. new version is in https://github.com/pytorch/xla/pull/3920/files#diff-17ef64dc5ef931bed0cb0d971cf537a610e90a9b4351658d6587ff59448f0f87, I don't think it will be too relevant to you guys. It is mostly updated because we need to wrap entry hlo compuation sometimes.

Reply:
Let's remove it for now then, we can reimplement it and test it again once we transitioned to functionization.

Reply:
@wonjoolee95 Just a heads up.